Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -869,6 +881,40 @@ private string GetZipDeployPathInfo()
             return pathmsg;
         }
 
+        private void BackupErrorZip(ArtifactDeploymentInfo zipDeploymentInfo)
+        {
+            try
+            {
+                // File created in D:\home\data\SitePackages for current deployment
+                var filePath = Path.Combine(_environment.SitePackagesPath, zipDeploymentInfo.ArtifactFileName);
+
+                // Store only if zip file size < 100 MB
+                long fileBytes = new FileInfo(filePath).Length;
+                float fileMB = (float)fileBytes / (1024 * 1024);
```

Comment:
for new codes, start practicing _024 for readability.

Reply:
Thanks. Added this change.